background.js

```diff
@@ -1,28 +1,36 @@
 // triggered when user clicks on installed extention icon
-chrome.browserAction.onClicked.addListener(function(tab) {
-    
-    // retrive domain from active tab
-    var url = tab.url;
-    var matches = url.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i);
-    var d = matches && matches[1].replace('www.','');
-    d = '.'+d;
+chrome.browserAction.onClicked.addListener(function (tab) {
 
-    // get all cookies for domain
-    chrome.cookies.getAll({domain: d}, function (cookies) {
+    let url = tab.url;
 
-        // iterate on cookie to get cookie detail
-        for (var i=0; i<cookies.length; i++) {
-            var url = "http" + (cookies[i].secure ? "s" : "") + "://" + cookies[i].domain + cookies[i].path;
-            var cname = cookies[i].name;
+    // clear all cookies for the current url
+    chrome.cookies.getAll({ url: url }, function (cookies) {
+        console.log("Clearing cookies for active urL", url, cookies);
+        clearCookies(cookies);
+    });
 
-            // delete cookie
-            chrome.cookies.remove({
-                "url": url,
-                "name": cname
-            });
-        }
+    // retrive domain from active tab
+    let matches = url.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i);
+    let domain = matches && matches[1].replace('www.', '');
+    domain = "." + domain;
 
-        // reload currect active tab
-        chrome.tabs.reload();
+    // get all cookies for domain
+    chrome.cookies.getAll({ domain: domain }, function (cookies) {
+        console.log("Clearing cookies for domain", domain, cookies);
+        clearCookies(cookies);
     });
 });
+
+function clearCookies(cookies) {
+    // iterate on cookie to get cookie detail
+    for (let i = 0; i < cookies.length; i++) {
+        let url = "http" + (cookies[i].secure ? "s" : "") + "://" + cookies[i].domain + cookies[i].path;
+        let cname = cookies[i].name;
+
+        // delete cookie
+        chrome.cookies.remove({
+            url: url,
+            name: cname
+        });
+    }
+}
```

Comment:
This method is identical the previous callback, but with `let` instead of `var`

Reply:
how can variables declare with `var` cause the issue?

Reply:
Sorry - I didn't mean to imply that `var` was the problem, just that I moved this method **and** changed `var` to `let`.  I will change it back if you'd like - the reason I changed it was because I added some variables with `let`, at which point it was inconsistent, so I updated to `let` because I thought it was the best practice.

Reply:
Thanks for explanation here @zastrowm